superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -18,6 +19,48 @@ const ROW_LIMIT_OPTIONS = [10, 50, 100, 250, 500, 1000, 5000, 10000, 50000];
 
 const SERIES_LIMITS = [0, 5, 10, 25, 50, 100, 500];
 
+const metricRenderer = m => (
```

Comment:
Agreed, I thought about it when starting to write it, especially makes sense as it is fairly large. I'll address that.